doc/user/content/sql/create-source/postgres.md

```diff
@@ -45,70 +47,46 @@ For this reason, the upstream database must be configured to support logical rep
 
 #### Creating a source
 
-To avoid creating multiple replication slots upstream and minimize the required bandwidth, Materialize ingests the raw replication stream data for **all** tables included in a specific publication. This means that, when you define a source:
+To avoid creating multiple replication slots upstream and minimize the required bandwidth, Materialize ingests the raw replication stream data for **all** tables included in a specific publication. When you define a Postgres source:
 
 ```sql
 CREATE SOURCE mz_source
-FROM POSTGRES
-  CONNECTION pg_connection
-  (PUBLICATION 'mz_source');
+  FROM POSTGRES
+    CONNECTION pg_connection
+    (PUBLICATION 'mz_source')
+  FOR ALL TABLES
+  WITH (SIZE '3xsmall');
 ```
 
-, its schema looks like:
+, Materialize will automatically create a **subsource** for each original table in the publication:
```

Comment:
This leading comma does not look right to my eye:

![image](https://user-images.githubusercontent.com/882976/195507952-a65bfc64-a6a2-4f6a-8922-1a4a7946e193.png)


```suggestion
Materialize will automatically create a **subsource** for each original table in the publication:
```